package.json

```diff
@@ -8,8 +8,8 @@
 	"author": "Cheqd Foundation Limited (https://github.com/cheqd)",
 	"repository": "https://github.com/cheqd/credential-service.git",
 	"scripts": {
-		"build": "webpack",
-		"start": "miniflare dist/worker.js --global-async-io --global-timers --global-random",
+		"build": "tsc",
```

Comment:
If you're not using TypeScript, then we can change this but it seems kinda weird to do a TypeScript compile and then use the index.js source file instead (since we're copying from `dist/src` in Dockerfile rather than the compiled output.